DittoHeartbeat/src/main/java/live.ditto.dittoheartbeat/Heartbeat.kt

```diff
@@ -95,12 +95,10 @@ fun addToCollection(info: DittoHeartbeatInfo, config: DittoHeartbeatConfig, ditt
         "metaData" to metaData,
         "sdk" to info.sdk,
         "_schema" to info.schema,
-        "peerKey" to info.peerKey
+        "pk" to info.peerKey
```

Comment:
Is this change intentional? I think we settled on spelled out `peerKey` for the name here